### PR TITLE
IndustryObject::drawIndustry

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Found this little func hadn't been done yet.

### CODE REVIEW

File: src/OpenLoco/Objects/IndustryObject.h [hunk 1]
```diff
@@ -51,16 +51,16 @@ namespace OpenLoco
         uint16_t nameSingular;        // 0x0A
         uint16_t namePlural;          // 0x0C
         uint32_t var_0E;
-        uint32_t var_12;
+        uint32_t var_12; // Base image id for building 0
```
Comment (LeftofZen) [inline src/OpenLoco/Objects/IndustryObject.h:54]: any reason not to rename the variable here (and below)?

Comment (duncanspumpkin) [reply src/OpenLoco/Objects/IndustryObject.h:54]: I'm not 100% sure at this time. I really need to do the paint function before being confident.

Comment (LeftofZen) [reply src/OpenLoco/Objects/IndustryObject.h:54]: sounds fine to me

**Review by LeftofZen — APPROVED** [review]
variables to be renamed after paint industry funcs are done

**Pull request merged**